### ISSUE TITLE
[BUG] Invalid csrf token on clean installation. 

### ISSUE THREAD

**Issue by Talref** [SillyTavern/SillyTavern#780]
I've searched for the bug but the solutions aren't applicable to me.

`ForbiddenError: invalid csrf token
    at doubleCsrf (/home/eradan/ai/SillyTavern/node_modules/csrf-csrf/lib/cjs/index.cjs:18:61)
    at Object.<anonymous> (/home/eradan/ai/SillyTavern/server.js:301:49)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47
ForbiddenError: invalid csrf token
    at doubleCsrf (/home/eradan/ai/SillyTavern/node_modules/csrf-csrf/lib/cjs/index.cjs:18:61)
    at Object.<anonymous> (/home/eradan/ai/SillyTavern/server.js:301:49)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47
`

Firefox on Ubuntu. I don't have any window/tab open when I launch SillyTavern. I tried a clean install and switch branch to no avail. The problem persists.

**Comment by cody-schultz-DS** [comment]
Please indicate how this issue was resolved. I am having the same issue. I broke some of the user settings attempting to make a directory clone of it with symbolic links for all the files that individual installations don't modify, however, I botched one of the settings and even after a fresh install this error continues to occur.

**Comment by Talref** [comment]
Sadly, I don't recall. But as I closed it immediately without any comment it was something silly, for sure. Try checking if you're in a venv, permissione or if you're using the right launcher.

**Comment by Talref** [comment]
update node too.

**Comment by cody-schultz-DS** [comment]
I did end up figuring it out, and yeah, it was silly. a CSRF token is made to prevent website spoofing essentially, so I had a cookie from my previous Sillytavern install and when I reinstalled it and went to the new one, it refused to connect because it noticed that it wasn't the same instance.

TLDR; you can fix this issue by clearing your cache on the client side.
